python/ray/tune/examples/ax_example.py

```diff
@@ -71,15 +68,17 @@ def easy_objective(config):
     }
     algo = AxSearch(
         max_concurrent=4,
-        metric="hartmann6",
-        mode="min",
         parameter_constraints=["x1 + x2 <= 2.0"],  # Optional.
         outcome_constraints=["l2norm <= 1.25"],  # Optional.
     )
-    scheduler = AsyncHyperBandScheduler(metric="hartmann6", mode="min")
-    tune.run(
+    scheduler = AsyncHyperBandScheduler()
+    analysis = tune.run(
         easy_objective,
         name="ax",
+        metric="hartmann6",  # provided in the 'easy_objective' function
+        mode="min",
         search_alg=algo,
         scheduler=scheduler,
         **tune_kwargs)
+
```

Comment:
I'm not a huge fan of the `**tune_kwargs` thing. Can we just use a config dict and pass the other kwargs directly as arguments?

Reply:
yep!